src/fmeval/eval_algorithms/faithfulness.py

```diff
@@ -0,0 +1,297 @@
+import logging
+from typing import Optional, List, Dict, Any
+
+from fmeval.constants import (
+    DatasetColumns,
+    MEAN,
+)
+from fmeval.data_loaders.util import get_dataset
+from fmeval.data_loaders.data_config import DataConfig
+from fmeval.eval_algorithms.save_strategy import SaveStrategy
+from fmeval.eval_algorithms.util import get_dataset_configs
+from fmeval.eval_algorithms.common import evaluate_dataset, get_default_judge_model
+from fmeval.eval_algorithms.eval_algorithm import EvalAlgorithmInterface, EvalAlgorithmConfig
+from fmeval.eval_algorithms import (
+    EvalAlgorithm,
+    EvalOutput,
+    EvalScore,
+)
+from fmeval.eval_algorithms.util import validate_dataset
+from fmeval.model_runners.model_runner import ModelRunner
+from fmeval.transforms.common import GetModelOutputs, GeneratePrompt
+from fmeval.transforms.transform import Transform
+from fmeval.transforms.transform_pipeline import TransformPipeline
+from fmeval.transforms.util import validate_call
+from fmeval.util import get_eval_results_path
+
+logger = logging.getLogger(__name__)
+
+FAITHFULNESS = EvalAlgorithm.FAITHFULNESS.value
+
+
+LONG_FORM_ANSWER_PROMPT = """\
```

Comment:
This prompt does not have any few shot (question, answer, statements) examples to demonstrate the expected output format to the judge model. Can you craft a few-shot prompt based on [NLI_STATEMENTS_MESSAGE] (https://github.com/xiaoyi-cheng/fmeval/blob/e49a89ab169055a68743d6af9def36c708854f4a/src/fmeval/eval_algorithms/faithfulness.py#L39)?